Dockerfile-demo

```diff
@@ -1,6 +1,8 @@
 FROM netenglabs/suzieq:latest
 
-COPY ./parquet /suzieq/parquet
+ARG username=suzieq
+
+COPY ./tests/data/parquet /home/$username/parquet
```

Comment:
I think this Dockerfile doesn't need any modification, having the parquet data in the user directory is wrong, since the config file points to the /suzieq directory

Reply:
I have this since I don't expect the user to do anything in the demo.